src/lib.rs

```diff
@@ -227,7 +227,7 @@
 pub mod client;
 /// Module representing the [documents] structures.
 pub mod documents;
-/// Module containing the [document::Document] trait.
+/// Module containing the [documents::Document] trait.
```

Comment:
> I changed 'document' to 'documents', which allows it to be compiled without warning, but I am not sure whether or not the dumps module really have documents::Document trait.

Absolutely haha, can you update it to mention the dumps?

Reply:
Sorry :< My English is poor so I can't understand what do you mean by `Absolutely`.
Should I keep the `documents` or just remove it and make it mention the dumps?

Reply:
Just  remove it and make it mention the dumps :)

Reply:
I have finished it